setup.cfg

```diff
@@ -80,5 +80,5 @@ depends =
     py{37,38,39,310,311,312}
 
 [flake8]
```

Comment:
Flake8? What is this 2012? 😂

Reply:
It's a bit of a [pascal's mugging](https://en.wikipedia.org/wiki/Pascal%27s_mugging) but I'm only migrating things using advice (or skel changes) now.  Feel free to send a PR moving this to ruff.